tools/make-tarball.sh

```diff
@@ -29,11 +29,11 @@ $(pwd)/tools/make-tarball.sh checkout $TMPDIR/$DIR/
 (
   # Remove unnessecery files
   cd $TMPDIR/$DIR
-  rm -fr appveyor.yml circle.yml .npmignore tools/ package.json emcc/
+  rm -fr appveyor.yml circle.yml .npmignore tools/*.{sh,py} tools/homebrew package.json emcc/
 )
 
 (
   cd $TMPDIR
-  echo "Creating tarball"
+  echo "Creating tarball $TARBALL"
```

Comment:
This makes it so that the message tells you where the tarball was created otherwise you're left guessing.